types/api/v1beta1/event.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+message Event {
+    int64 timestamp = 1; // google timestamp
+    uint32 id = 2;
+    string name = 3;
+    Policies policies = 4;
+    Context context = 5;
+    repeated EventData data = 6;
+    optional Threat threat = 7;
+}
+
+message Policies {
+    repeated string matched = 1;
+}
+
+message Context {
+    optional Process process = 1;
+    optional Container container = 2;
+    optional K8S k8s = 3;
+}
+
+message Process  {
+    Executable executable = 1;
+    int64 pid = 2;
+    int32 namespaced_pid = 3;
```

Comment:
shouldn't be of the same type as namespaced_pid?

Reply:
both will be uint32